front/components/assistant/conversation/AgentMessage.tsx

```diff
@@ -619,29 +619,20 @@ export function AgentMessage({
 
         {agentMessage.content !== null && (
           <div>
-            {lastTokenClassification !== "chain_of_thought" &&
-            agentMessage.content === "" ? (
-              <div className="blinking-cursor">
-                <span></span>
-              </div>
-            ) : (
-              <CitationsContext.Provider
-                value={{
-                  references,
-                  updateActiveReferences,
-                }}
-              >
-                <Markdown
-                  content={sanitizeVisualizationContent(agentMessage.content)}
-                  isStreaming={
-                    streaming && lastTokenClassification === "tokens"
-                  }
-                  isLastMessage={isLastMessage}
-                  additionalMarkdownComponents={additionalMarkdownComponents}
-                  additionalMarkdownPlugins={additionalMarkdownPlugins}
-                />
-              </CitationsContext.Provider>
-            )}
+            <CitationsContext.Provider
```

Comment:
IIUC we don't need to add any condition to render markdown, it will be an empty string and we just render nothing 🤔